code/game/machinery/portable_turret.dm

```diff
@@ -264,53 +264,43 @@ var/global/list/turret_icons
 		if(isCrowbar(I))
 			//If the turret is destroyed, you can remove it with a crowbar to
 			//try and salvage its components
-			to_chat(user, "<span class='notice'>You begin prying the metal coverings off.</span>")
+			to_chat(user, "You begin prying the metal coverings off.")
 			if(do_after(user, 20, src))
 				if(prob(70))
-					to_chat(user, "<span class='notice'>You remove the turret and salvage some components.</span>")
+					to_chat(user, "You remove the turret and salvage some components.")
 					physically_destroyed()
 				else
-					to_chat(user, "<span class='notice'>You remove the turret but did not manage to salvage anything.</span>")
+					to_chat(user, "You remove the turret but did not manage to salvage anything.")
 
 	else if(isWrench(I))
 		if(enabled || raised)
-			to_chat(user, "<span class='warning'>You cannot unsecure an active turret!</span>")
+			to_chat(user, "<span class='warning'>You cannot unsecure an active turret!")
 			return
 		if(wrenching)
-			to_chat(user, "<span class='warning'>Someone is already [anchored ? "un" : ""]securing the turret!</span>")
+			to_chat(user, "<span class='warning'>Someone is already [anchored ? "un" : ""]securing the turret!")
 			return
 		if(!anchored && isspaceturf(get_turf(src)))
-			to_chat(user, "<span class='warning'>Cannot secure turrets in space!</span>")
+			to_chat(user, "<span class='warning'>Cannot secure turrets in space!")
```

Comment:
You missed a few initial spans in this file. Also I feel like they should still be warnings.